content/blog/0012-effect-3.0.mdx

```diff
@@ -0,0 +1,350 @@
+  httpCall(id).pipe(
+    Effect.timeout("1 seconds"),
+    Effect.retry(retryPolicy),
+    Effect.withSpan("getTodo", { attributes: { id } })
+  )
+```
+
+In this example we can truly see the power of composition, each block of Effect cares about a specific thing (such as schedule definitions, http client, etc) and it does so in such a way that the single pieces compose together.
+
+The last problem we wanted to solve with our code is checking the types at the edge, accounting for that our full example code becomes:
+
+```ts
+import { HttpClient } from "@effect/platform"
+import { Schema } from "@effect/schema"
+import { Effect, Schedule } from "effect"
+
+const retryPolicy = Schedule.exponential(1000).pipe(
+  Schedule.compose(Schedule.recurs(3))
+)
+
+const httpCall = (id: number) =>
+  HttpClient.request.get(`/todos/${id}`).pipe(
+    HttpClient.client.fetchOk(),
+    HttpClient.response.json
+  )
+
+class Todo extends Schema.Class<Todo>("Todo")({
+  id: Schema.number,
+  userId: Schema.number,
+  title: Schema.string,
+  completed: Schema.boolean
+}) {}
+
+export const getTodo = (id: number) =>
+  httpCall(id).pipe(
+    Effect.timeout("1 seconds"),
+    Effect.retry(retryPolicy),
+    Effect.flatMap(Schema.decodeUnknown(Todo)),
+    Effect.withSpan("getTodo", { attributes: { id } })
+  )
+```
+
+## The Future
+
+While the core `effect` library is now stable, the rest of the ecosystem such as `@effect/platform` and `@effect/schema` aren't yet. It will be our first priority to make the ecosystem libraries stable (3-6 months), together with adding tons of documentation and examples.
+
+Following that, we plan to keep iterating and develop higher and higher levels of abstraction to solve challenging issues in the development of Production-Grade TypeScript. 
+
+Our next goal for the near future (6-12 months) is to build up [Effect Cluster](https://github.com/Effect-TS/cluster), the first real JavaScript solution that enables:
+- Clustering of Distributed Instances
+- Addressing of Processes by Name
+- Actors and Entities
+- Scheduling of Cluster Singletons
+- Execution of Durable Business Workflows
+
+## The Company
+
+A year ago with the objective of making Effect as easy to use and as feature complete as possible we:
+- Incorporated [Effectful Technologies Inc](https://effectful.co/).
+- Raised a Seed Round of $redacted$ from $redacted$. 
+- Hired an amazing Founding Team.
+
+As a VC-backed company that deals with Free Open Source Software, especially in recent times, we are aware of our responsibility of being absolutely clear about what is OSS and what isn't.
+
+In our case everything that is released under MIT license will remain MIT licensed, in fact in order to contribute we don't even require CLAs and the Effect organization is fully managed by the Community.
+
+To make money we will build amazing commercial products and services where we leverage Effect to its fullest power, and you'll have the choice of using them as you see fit.
```

Comment:
This paragraph is a tad crass, maybe something phrased like
> "We will never charge for using effect or its ecosystem, nor will we sell 'premium' libs (<- idk, never came up, just wishful thinking on my part). Effectful Technologies will house and build its own effect-powered products and services that you can choose using as you see fit."

Will show more conviction on this matter.
I expect the people who read these line and need the reassurance to start-off with an overly-skeptical approach so clear, dry language might be better than an excited tone?